Docs/Manual Build Instructions.md

```diff
@@ -0,0 +1,30 @@
+# Manual Build Instructions for the Data Sharing Frameowrk
+These steps will allow you to manually build all of the components necessary to use the Data Sharing Framework Custom Device. This is not recommended for end-users of the Custom Device; the [Releases page](https://github.com/ni/niveristand-data-sharing-framework-custom-device/releases) should be used to find the latest builds of the Custom Device for supported versions of VeriStand.
+
+For each Target type:
+
+1.	Build DSF Core PPL
+	1. 	Open the target build project in [repo root]\Core
+	1. 	Execute the build spec
+	1. 	The Core PPL will be output to C:\Builds\Data Sharing Framework\<Target>\Core
```

Comment:
Does it still output to `C:\Builds`?

Reply:
No. the built PPL outputs to ../Built/Core/<target>/DSF Core.lvlibp

Reply:
I didn't re-write the content. Mostly just formatted it. 
Let me update this with all the finalized build layout changes, then I'll put it back up for review.